### DISCUSSION TITLE
Fleetmon errors

### DISCUSSION THREAD

**Discussion by dziban303** [jvde-github/AIS-catcher#44] [General]
Hey, regarding the Fleetmon errors mentioned in the 0.41 patch notes, what percentage are you seeing being reported as errors? Is there a stats page I'm not seeing that shows these values enumerated? All I can find on my account is a graph, where only the very top few pixels are colored yellow for errors.

I notice that the Fleetmon station stats are showing strange, obviously–wrong values when changing the interval of the charts. Like my 1 year stats showing a total of ~540 hours online while my 30-day stats show 700 hours online; another howler is an average of 12,000 messages per day in the past week, but 140,000 messages per day in the last day. Nothing in the graphs themselves support these wildly differing numbers. This tells me the stats calculation is rather ill. Can any of these statistics be taken seriously?

Cheers

**Comment by jvde-github** [comment]
Hi! I mentioned it only because looking at the graph led to the idea of the filter, also because many sites throw out a lot of message types anyway. 

But it is quite sizeable (20% or so) but fairly constant. See the picture at the bottom, you can see my normal station output, then at some point I started to filter and send only type 1,2,3,5,18,19,24 at which point the orange disappears. Next, only type 4 is send (which is a constant rate almost with constant portion of errors) and then back to normal. So my conclusion was that part of the type 4 msgs are classified as "error". I think this is from a base station despite the fact that the messages are absolutely fine (just not from a vessel).

You are right on the stats, did not notice it before! I am 18 hours online this year and 24 hours yesterday.... This part of the site needs some TLC, I guess. I like the weather stats though.

Cheers

![fleetmon stats](https://user-images.githubusercontent.com/52420030/204613616-da53c0e4-6112-4097-8fd5-2147624c50cc.jpg)
